get-customer/src/main/java/no/unit/nva/customer/get/GetCustomerHandler.java

```diff
@@ -17,7 +17,7 @@
 
 import static org.apache.http.HttpStatus.SC_OK;
 
-public class GetCustomerHandler extends ApiGatewayHandler<Customer,Customer> {
+public class GetCustomerHandler extends ApiGatewayHandler<String,Customer> {
```

Comment:
What do we set when we have no input data?

Reply:
In the current version String. But maybe we fix this